development/docker-compose.base.yml

```diff
@@ -21,8 +21,9 @@ services:
         condition: "service_started"
       db:
         condition: "service_healthy"
-    <<: *nautobot-build
-    <<: *nautobot-base
+    <<:
+      - *nautobot-base
+      - *nautobot-build
```

Comment:
```suggestion
    <<: [*nautobot-base, *nautobot-build]
```

Reply:
This is what most other projects have.

Reply:
nornir, ssot-arista-cloudvision, ssot-servicenow and capacity metrics use `[*nautobot-base, *nautobot-build]` while all the rest use what is already in the PR.